src/Features/Core/Portable/Completion/CompletionRules.cs

```diff
@@ -20,20 +21,21 @@ internal class CompletionRules
         private readonly object _gate = new object();
         private readonly AbstractCompletionService _completionService;
         private readonly Dictionary<string, PatternMatcher> _patternMatcherMap = new Dictionary<string, PatternMatcher>();
+        private readonly Dictionary<string, PatternMatcher> _invariantCulturePatternMatcherMap = new Dictionary<string, PatternMatcher>();
 
         public CompletionRules(AbstractCompletionService completionService)
         {
             _completionService = completionService;
         }
 
-        protected PatternMatcher GetPatternMatcher(string value)
+        protected PatternMatcher GetPatternMatcher(string value, CultureInfo culture)
```

Comment:
I don't believe TS calls this.  I searched here: https://github.com/Microsoft/TypeScript-VisualStudio/search?utf8=%E2%9C%93&q=GetPatternMatcher  and did not find it.